scan/cli.go

```diff
@@ -350,7 +354,9 @@ func DockerScan(c *cli.Context, image string) error {
 		SetIncludeLicenses(c.Bool("licenses")).
 		SetFail(c.BoolT("fail")).
 		SetPrintExtendedTable(c.Bool(cliutils.ExtendedTable)).
-		SetBypassArchiveLimits(c.Bool(cliutils.BypassArchiveLimits))
+		SetBypassArchiveLimits(c.Bool(cliutils.BypassArchiveLimits)).
+		SetFixableOnly(c.Bool(cliutils.FixableOnly)).
+		SetMinSeverityFilter(c.String(cliutils.MinSeverity))
```

Comment:
It's better to validate the MinSeverityFilter like we do in format:
format, err := commandsutils.GetXrayOutputFormat(c.String("format"))

Reply:
done